### DISCUSSION TITLE
5.0版本如何设置节点中label的字体大小

### DISCUSSION THREAD

**Discussion by needones** [antvis/G6#6204] [Q&A]
5.0版本如何设置节点中label的字体大小，4.0版本有这个设置，我看了5.0版本的文档并没有找到相关的描述，
https://g6-next.antv.antgroup.com/api/elements/nodes/base-node

请问这个版本还可以设置字体大小吗？

**Comment by resetsix** [comment]
> 5.0版本如何设置节点中label的字体大小，4.0版本有这个设置，我看了5.0版本的文档并没有找到相关的描述， https://g6-next.antv.antgroup.com/api/elements/nodes/base-node
> 
> 请问这个版本还可以设置字体大小吗？

```tsx
const graph = new Graph({
  container: xxx,
  nodes:{
    style:{
      // 在 v5 中，label 变成了布尔值，labelText才是你的文本
      labelText:"等同于 v4 的 label"
      labelFontSize:"这是你想要的字体大小配置"
    }
  })
```
